test/deviceplugin/pkg/numacell/cmd/numacell.go

```diff
@@ -38,9 +38,17 @@ import (
 func summarize(topoInfo *topology.Info) string {
 	var buf strings.Builder
 	for _, node := range topoInfo.Nodes {
-		fmt.Fprintf(&buf, "NUMA node %d\n", node.ID)
+		_, err := fmt.Fprintf(&buf, "NUMA node %d\n", node.ID)
+		if err != nil {
+			klog.Warningf("failed writing data to builder: %v\n", err)
+			return buf.String()
+		}
 		for _, core := range node.Cores {
-			fmt.Fprintf(&buf, "\t%s\n", core.String())
+			_, err := fmt.Fprintf(&buf, "\t%s\n", core.String())
+			if err != nil {
+				klog.Warningf("failed writing data to builder: %v\n", err)
+				return buf.String()
+			}
```

Comment:
this can't really happen. If it does the sky ha fell already. I'd ignore this specific linter check.

Reply:
I thought since this is printed out to a returned value then it must be important to check. I skipped fmt.Fprintf for this linter